addons/l10n_in/models/account_invoice.py

```diff
@@ -542,6 +542,16 @@ def _get_name_invoice_report(self):
 
     def _post(self, soft=True):
         """Use journal type to define document type because not miss state in any entry including POS entry"""
+        if any(move.company_id.chart_template == 'in' and move.country_code != 'IN' for move in self):
+            action = self.env.ref('account.action_account_config')
+            raise RedirectWarning(
+                message=self.env._(
+                    "The Fiscal country and your fiscal localization's country do not match.\n"
+                    "Please change it from your Account's configuration.",
+                ),
+                action=action.id,
+                button_text=self.env._('Go to settings'),
```

Comment:
```suggestion
                button_text=self.env._("Go to settings"),
```
Define translator(`_`) rather than calling `self.env._` everytime